apps/project/schema.py

```diff
@@ -0,0 +1,63 @@
+import graphene
+from django.db.models import QuerySet
+from graphene_django import DjangoObjectType
+from graphene_django_extras import DjangoObjectField, PageGraphqlPagination
+
+from utils.graphene.types import CustomDjangoListObjectType
+from utils.graphene.fields import DjangoPaginatedListObjectField
+from lead.schema import Query as LeadQuery
+
+from .models import Project
+from .filter_set import ProjectFilterSet
+
+
+class ProjectType(
+    # -- Start --Project scopped entities
+    LeadQuery,
+    # --  End  --Project scopped entities
+    DjangoObjectType,
+):
+    class Meta:
+        model = Project
+        fields = (
+            'id', 'title', 'description', 'start_date', 'end_date',
+            'members', 'regions', 'user_groups', 'analysis_framework',
+            'category_editor', 'assessment_template', 'data',
+            'is_default', 'is_private', 'is_visualization_enabled', 'status',
+            'organizations', 'stats_cache',
+        )
+
+    current_user_role = graphene.String()
+
+    # NOTE: This is a custom feature
+    # see: https://github.com/eamigo86/graphene-django-extras/compare/graphene-v2...the-deep:graphene-v2
+    @staticmethod
+    def get_custom_node(queryset, info, id):
+        try:
+            project = Project.get_for_gq(info.context.user).get(pk=id)
+            info.context.set_active_project(project)
+            return project
+        except Project.DoesNotExist:
+            return None
```

Comment:
What does this method do?

Reply:
There is a comment there.

    # NOTE: This is a custom feature
    # see: https://github.com/eamigo86/graphene-django-extras/compare/graphene-v2...the-deep:graphene-v2